test/tests/functions/date.json

```diff
@@ -11,7 +11,7 @@
       "exp": "{:date}",
       "expErrors": [
         {
-          "type": "invalid-expression"
+          "type": "bad-operand"
```

Comment:
Just highlighting an assumption I've made:
This test results in `bad-operand` because it is effectively being passed an operand of 'undefined', which is not compatible with the `:date` function.